bindings/nostr-sdk-ffi/src/client/mod.rs

```diff
@@ -616,15 +615,11 @@ impl Client {
         &self,
         receiver: &PublicKey,
         rumor: Arc<EventBuilder>,
-        expiration: Option<Arc<Timestamp>>,
+        extra_tags: Vec<Tag>,
     ) -> Result<SendEventOutput> {
         Ok(self
             .inner
-            .gift_wrap(
-                receiver.deref(),
-                rumor.as_ref().deref().clone(),
-                expiration.map(|t| **t),
-            )
+            .gift_wrap(receiver.deref(), rumor.as_ref().deref().clone(), extra_tags)
```

Comment:
@yukibtc I've never dealt with bindings before. I'm not sure what needs to happen with the vec arg here but this isn't working currently.

Reply:
No problem, I'm going to fix it during the cherry-picking. In UniFFI bindings the objects must be often wrapped inside an `Arc`